routes/api.php

```diff
@@ -34,3 +35,6 @@
 
 Route::get('/patient/select', [PatientsController::class, 'select'])->middleware(['auth:api'])->name('patient.select');
 Route::get('/patient/vitals/{id}', [PatientsController::class, 'patientVital'])->middleware(['auth:api'])->name('patient.vital');
+
+Route::apiResource('/medication', MedicationController::class)->middleware(['auth:api']);
```

Comment:
you haven't use other methods like index, show, edit, update so please use the name of the method which are used only
store only here.